examples/tab_autocompletion.py

```diff
@@ -109,6 +109,18 @@ def instance_query_actors(self) -> List[str]:
         """Simulating a function that queries and returns a completion values"""
         return actors
 
+    def instance_query_movie_ids(self) -> List[str]:
+        """Demonstrates showing tabular hinting of tab completion information"""
+        completions_with_desc = []
+
+        for movie_id, movie_entry in self.MOVIE_DATABASE.items():
```

Comment:
This is a somewhat complicated feature, but a really nice one.  Please consider updating the Sphinx documentation to refer to it.

You continue to raise the bar of awesome semi-automatic tab-completion. 

You should consider making some sort of public tutorial of how to use all of the wonderful new features.  The examples are useful, but something that provides a little more hand-holding to introduce users to the new features would be amazing.  I'm not sure how discoverable some of the new features are - i.e. they are awesome and powerful, but how do we make sure new users know they exist and give them a basic clue about how to use them?